---
layout: paper
title: "Enhanced annealing of mismatched oligonucleotides using a novel melting curve assay allows efficient in vitro discrimination and restriction of a single nucleotide polymorphism"
year: "2011"
shortref: "Doyle <i>et al.</i> <i>BMC Biotechnology</i> 2011"
nickname: mismatch-oligos-melt-curves
journal: "BMC Biotechnology"
volume: 
issue:
pages: 
authors: "DOYLE SR, Chan CK, and Grant WN"
image: 
redirect_from: 
fulltext: https://bmcbiotechnol.biomedcentral.com/articles/10.1186/1472-6750-11-83
pdflink: 
github: 
pmid: 
pmcid: 
f1000: 
doi: "10.1186/1472-6750-11-83"
dryad_doi:
figshare_doi: 
zenodo_doi: 
altmetric_id: 
category: paper
# Note: 'published' is a Jekyll keyword and does not refer to whether the paper is published, but rather to whether this Markdown should be part of the rendered site.
published: true
preprint: true
embargo: false	
peerreview: false
review: false
tags: [SNPs, discrimination, mismatch oligos, melt curve]
---
{% include JB/setup %}

# Abstract 

**Background**
Many SNP discrimination strategies employ natural restriction endonucleases to discriminate between allelic states. However, SNPs are often not associated with a restriction site and therefore, a number of attempts have been made to generate sequence-adaptable restriction endonucleases. In this study, a simple, sequence-adaptable SNP discrimination mechanism between a 'wild-type' and 'mutant' template is demonstrated. This model differs from other artificial restriction endonuclease models as cis- rather than trans-orientated regions of single stranded DNA were generated and cleaved, and therefore, overcomes potential issues of either inefficient or non-specific binding when only a single variant is targeted.

**Results**
A series of mismatch 'bubbles' that spanned 0-5-bp surrounding a point mutation was generated and analysed for sensitivity to S1 nuclease. In this model, generation of oligonucleotide-mediated ssDNA mismatch 'bubbles' in the presence of S1 nuclease resulted in the selective degradation of the mutant template while maintaining wild-type template integrity. Increasing the size of the mismatch increased the rate of mutant sequence degradation, until a threshold above which discrimination was lost and the wild-type sequence was degraded. This level of fine discrimination was possible due to the development of a novel high-resolution melting curve assay to empirically determine changes in Tm (~5.0°C per base-pair mismatch) and to optimise annealing conditions (~18.38°C below Tm) of the mismatched oligonucleotide sets.

**Conclusions**
The *in vitro* 'cleavage bubble' model presented is sequence-adaptable as determined by the binding oligonucleotide, and hence, has the potential to be tailored to discriminate between any two or more SNPs. Furthermore, the demonstrated fluorometric assay has broad application potential, offering a rapid, sensitive and high-throughput means to determine Tm and annealing rates as an alternative to conventional hybridisation detection strategies.


