---
layout: paper
title: "A Genome Resequencing-Based Genetic Map Reveals the Recombination Landscape of an Outbred Parasitic Nematode in the Presence of Polyploidy and Polyandry"
year: "2018"
shortref: "Doyle  <i>et al.</i> <i>GBE</i> 2018"
nickname: hcontortus-genetic-map
journal: "Genome Biology and Evolution"
volume: 10
issue: 2
pages: 396â€“409
authors: "DOYLE SR, Laing R, Bartley DJ, Britton C, Chaudhry U, Gilleard JS, Holroyd N, Mable BK, Maitland K, Morrison AA, Tait A, Tracey A, Berriman M, Devaney E, Cotton JA, Sargison ND"
image: /assets/images/papers/2018-02-01-hcontortus-genetic-map.png
redirect_from: 
fulltext: https://academic.oup.com/gbe/article/10/2/396/4756466
pdflink: 
github: 
pmid: 
pmcid: 
f1000: 
doi: "10.1093/gbe/evx269"
dryad_doi:
figshare_doi: 
zenodo_doi: 
altmetric_id: 
category: paper
# Note: 'published' is a Jekyll keyword and does not refer to whether the paper is published, but rather to whether this Markdown should be part of the rendered site.
published: true
preprint: true
embargo: false	
peerreview: false
review: false
tags: [hcontortus, genetic map, polyandry, polyploidy]
---
{% include JB/setup %}

# Abstract 

The parasitic nematode *Haemonchus contortus* is an economically and clinically important pathogen of small ruminants, and a model system for understanding the mechanisms and evolution of traits such as anthelmintic resistance. Anthelmintic resistance is widespread and is a major threat to the sustainability of livestock agriculture globally; however, little is known about the genome architecture and parameters such as recombination that will ultimately influence the rate at which resistance may evolve and spread. Here, we performed a genetic cross between two divergent strains of *H. contortus*, and subsequently used whole-genome resequencing of a female worm and her brood to identify the distribution of genome-wide variation that characterizes these strains. Using a novel bioinformatic approach to identify variants that segregate as expected in a pseudotestcross, we characterized linkage groups and estimated genetic distances between markers to generate a chromosome-scale F1 genetic map. We exploited this map to reveal the recombination landscape, the first for any helminth species, demonstrating extensive variation in recombination rate within and between chromosomes. Analyses of these data also revealed the extent of polyandry, whereby at least eight males were found to have contributed to the genetic variation of the progeny analyzed. Triploid offspring were also identified, which we hypothesize are the result of nondisjunction during female meiosis or polyspermy. These results expand our knowledge of the genetics of parasitic helminths and the unusual life-history of *H. contortus*, and enhance ongoing efforts to understand the genetic basis of resistance to the drugs used to control these worms and for related species that infect livestock and humans throughout the world. This study also demonstrates the feasibility of using whole-genome resequencing data to directly construct a genetic map in a single generation cross from a noninbred nonmodel organism with a complex lifecycle.

# Data Availability

The raw sequencing data generated and/or analyzed during this study are available in the European Nucleotide Archive repository, http://www.ebi.ac.uk/ena/ under the study accession number ERP024253. The genome assembly is available at ftp://ngs.sanger.ac.uk/production/pathogens/Haemonchus_contortus.


