---
layout: paper
title: "Population genomic and evolutionary modelling analyses reveal a single major QTL for ivermectin drug resistance in the pathogenic nematode, <i>Haemonchus contortus</i>"
year: "2019"
shortref: "Doyle  <i>et al.</i> <i>BMC Genomics</i> 2019"
nickname: hcontortus_ivm_backcross
journal: "BMC Genomics"
volume: 
issue: 20
pages: 218
authors: "DOYLE SR, Illingworth CJR, Laing R, Bartley DJ, Redman E, Martinelli A, Holroyd N, Morrison AA, Rezansoff A, Tracey A, Devaney E, Berriman M, Sargison N, Cotton JA, Gilleard JS"
image: /assets/images/papers/2019-03-15-hcontortus-ivm-backcross.png
redirect_from: 
fulltext: https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-019-5592-6
pdflink: 
github: 
pmid: 
pmcid: 
f1000: 
doi: "10.1186/s12864-019-5592-6"
dryad_doi:
figshare_doi: 
zenodo_doi: 
altmetric_id: 
category: paper
# Note: 'published' is a Jekyll keyword and does not refer to whether the paper is published, but rather to whether this Markdown should be part of the rendered site.
published: true
preprint: true
embargo: false	
peerreview: false
review: false
tags: [hcontortus, backcross, ivermectin, resistance]
---
{% include JB/setup %}

# Abstract 

**Background**
Infections with helminths cause an enormous disease burden in billions of animals and plants worldwide. Large scale use of anthelmintics has driven the evolution of resistance in a number of species that infect livestock and companion animals, and there are growing concerns regarding the reduced efficacy in some human-infective helminths. Understanding the mechanisms by which resistance evolves is the focus of increasing interest; robust genetic analysis of helminths is challenging, and although many candidate genes have been proposed, the genetic basis of resistance remains poorly resolved.

**Results**
Here, we present a genome-wide analysis of two genetic crosses between ivermectin resistant and sensitive isolates of the parasitic nematode *Haemonchus contortus*, an economically important gastrointestinal parasite of small ruminants and a model for anthelmintic research. Whole genome sequencing of parental populations, and key stages throughout the crosses, identified extensive genomic diversity that differentiates populations, but after backcrossing and selection, a single genomic quantitative trait locus (QTL) localised on chromosome V was revealed to be associated with ivermectin resistance. This QTL was common between the two geographically and genetically divergent resistant populations and did not include any leading candidate genes, suggestive of a previously uncharacterised mechanism and/or driver of resistance. Despite limited resolution due to low recombination in this region, population genetic analyses and novel evolutionary models supported strong selection at this QTL, driven by at least partial dominance of the resistant allele, and that large resistance-associated haplotype blocks were enriched in response to selection.

**Conclusions**
We have described the genetic architecture and mode of ivermectin selection, revealing a major genomic locus associated with ivermectin resistance, the most conclusive evidence to date in any parasitic nematode. This study highlights a novel genome-wide approach to the analysis of a genetic cross in non-model organisms with extreme genetic diversity, and the importance of a high-quality reference genome in interpreting the signals of selection so identified.

# Data Availability

The raw sequencing data generated during the current study are available in the European Nucleotide Archive repository (http://www.ebi.ac.uk/ena/) under the study accession number PRJEB2353 (Additional file 12: Table S1). The reference genome assembly is available from ftp://ngs.sanger.ac.uk/production/pathogens/Haemonchus_contortus. Code used in this project is available from https://github.com/cjri/HCCross.


