---
layout: paper
title: "Genome-wide analysis of response to ivermectin treatment by a Swedish field population of <i>Haemonchus contortus</i>"
year: "2022"
shortref: "Baltrusis <i>et al.</i> <i>IJP-DDR</i> 2022"
nickname: ivm-sweden
journal: "IJP DDR"
volume: "18" 
issue:
pages: "12-19"
authors: "Baltrusis P, Doyle SR, Halvarsson P, Hoglund J "
image: /assets/images/papers/2021-12-20-ivm-sweden.png
redirect_from: 
fulltext: "https://www.sciencedirect.com/science/article/pii/S2211320721000555?via%3Dihub"
pdflink: 
github: "https://github.com/pauliusbaltrusis/WGanalysis"
pmid: 
pmcid: 
f1000: 
doi: "https://doi.org/10.1016/j.ijpddr.2021.12.002"
dryad_doi:
figshare_doi: 
zenodo_doi: 
altmetric_id: 
category: paper
# Note: 'published' is a Jekyll keyword and does not refer to whether the paper is published, but rather to whether this Markdown should be part of the rendered site.
published: true
preprint: false
embargo: false	
peerreview: true
review: false
tags: [hcontortus, drug resistance, genetic mapping, ivermectin resistance]
---
{% include JB/setup %}

# Abstract 

<i>Haemonchus contortus</i> is a pathogenic gastrointestinal nematode of small ruminants and, in part due to its capacity to develop resistance to drugs, 
contributes to significant losses in the animal production sector worldwide. Despite decades of research, comparatively little is known about the specific mechanism(s) driving resistance to drugs such as ivermectin in this species. Here we describe a genome-wide approach to detect evidence of selection by ivermectin treatment in a field population of H. contortus from Sweden, using parasites sampled from the same animals before and seven days after ivermectin exposure followed by whole-genome sequencing. Despite an 89% reduction in parasites recovered after treatment measured by the fecal egg count reduction test, the surviving population was highly genetically similar to the population before treatment, suggesting that resistance has likely evolved over time and that resistance alleles are present on diverse haplotypes. Pairwise gene and SNP frequency comparisons indicated the highest degree of differentiation was found at the terminal end of chromosome 4, whereas the most striking difference in nucleotide diversity was observed in a region on chromosome 5 previously reported to harbor a major quantitative trait locus involved in ivermectin resistance. These data provide novel insight into the genome-wide effect of ivermectin selection in a field population as well as confirm the importance of the previously established quantitative trait locus in the development of resistance to ivermectin.
