---
layout: paper
title: "Genomic landscape of drug response reveals novel mediators of anthelmintic resistance"
year: "2021"
shortref: "Doyle <i>et al.</i> <i>bioRxiv</i> 2021"
nickname: x-qtl
journal: "bioRxiv"
volume: 
issue:
pages: 
authors: "Doyle SR, Laing R, Bartley D, Morrison A, Holroyd N, Maitland K, Antonopoulos A, Chaudhry U, Flis I, Howell S, McIntyre J, Gilleard JS, Tait A, Mable BK, Kaplan R, Sargison N, Britton C, Berriman M, Devaney E, Cotton J"
image:
redirect_from: 
fulltext: https://www.biorxiv.org/content/10.1101/2021.11.12.465712v2.full.pdf
pdflink: 
github: https://github.com/stephenrdoyle/hcontortus_xqtl
pmid: 
pmcid: 
f1000: 
doi: "10.1101/2021.11.12.465712"
dryad_doi:
figshare_doi: 
zenodo_doi: 
altmetric_id: 
category: paper
# Note: 'published' is a Jekyll keyword and does not refer to whether the paper is published, but rather to whether this Markdown should be part of the rendered site.
published: true
preprint: true
embargo: false	
peerreview: false
review: false
tags: [hcontortus, x-qtl, drug resistance, genetic mapping]
---
{% include JB/setup %}

# Abstract 

Understanding the genetic basis of anthelmintic drug resistance in parasitic nematodes is key to improving the efficacy and sustainability of parasite control. Here, we use a genetic cross in a natural host-parasite system to simultaneously map resistance loci for the three major classes of anthelmintics. This approach identifies novel alleles for resistance to benzimidazoles and levamisole and implicates the transcription factor, *cky-1*, in ivermectin resistance. This gene is within a locus under selection in ivermectin resistant populations worldwide; functional validation using knockout and gene expression experiments supports a role for *cky-1* overexpression in ivermectin resistance. Our work demonstrates the feasibility of high-resolution forward genetics in a parasitic nematode, and identifies variants for the development of molecular diagnostics to combat drug resistance in the field.


## Data availability

Raw sequencing data for this study are outlined in table S1 and are archived under
the ENA study accession PRJEB4207. The *H. contortus* genome assembly and
manually curated annotation resources are publicly available at
https://parasite.wormbase.org/Haemonchus_contortus_prjeb506/Info/Index/. The
code used to generate and analyse data and to plot figures can be found at
https://github.com/stephenrdoyle/hcontortus_X-QTL.
