---
layout: paper
title: "The genome sequence of the Australian filarial nematode, Cercopithifilaria johnstoni"
year: "2021"
shortref: "McCann <i>et al.</i> <i>Wellcome Open Research</i> 2021"
nickname: cjohnstoni_genome
journal: "Wellcome Open Research"
volume: 
issue:
pages: 
authors: "McCann K, Grant W, Doyle SR"
image: /assets/images/papers/2021-10-12-cjohnstoni_genome_pic.png
redirect_from: 
fulltext: https://wellcomeopenresearch.org/articles/6-259/v1 
pdflink: 
github: https://github.com/stephenrdoyle/cercopithifilaria_johnstoni
pmid: 
pmcid: 
f1000: 
doi: "10.12688/wellcomeopenres.17258.1"
dryad_doi:
figshare_doi: 
zenodo_doi: "10.5281/zenodo.5545956"
altmetric_id: 
category: paper
# Note: 'published' is a Jekyll keyword and does not refer to whether the paper is published, but rather to whether this Markdown should be part of the rendered site.
published: true
preprint: true
embargo: false	
peerreview: false
review: false
tags: [cjohnstoni, genome, assembly, annotation]
---
{% include JB/setup %}

# Abstract 

We present a genome assembly and annotation of an individual female Cercopithifilaria johnstoni, a parasitic filarial nematode that is transmitted by hard ticks (Ixodidae) to infect a broad range of native Australian murid and marsupial hosts. The genome sequence is 76.9 Mbp in length, and although in draft form (N50 = 99 kbp, N50[n] = 232), is largely complete based on universally conserved orthologs (BUSCOs; genome = 94.9%, protein = 96.5%) and relative to other related filarial species. These data represent the first genomic resources for the genus Cercopithifilaria, a group of parasites with a broad host range, and form the basis for comparative analysis with the human-infective parasite, Onchocerca volvulus, both of which are responsible for similar eye and skin pathologies in their respective hosts.

## Data availability

European Nucleotide Archive: Raw sequence data, genome and annotation are deposited in the ENA. Accession number PRJEB47283; https://identifiers.org/ena.embl:PRJEB47283. The analysis code used in this study is available from GitHub and is archived with Zenodo (Doyle & McKann, 2021).
