---
layout: paper
title: "Transcriptomic analyses implicate neuronal plasticity and chloride homeostasis in ivermectin resistance and response to treatment in a parasitic nematode"
year: "2022"
shortref: "Laing <i>et al.</i> <i>PLoS Path</i> 2022"
nickname: x-qtl_rnaseq
journal: "PLoS Pathogens"
volume: 
issue:
pages: 
authors: "Laing R, Doyle SR, McIntyre J, Maitland K, Morrison A, Bartley DJ, Kaplan R, Chaudhry U, Sargison N, Tait A, Cotton JA, Britton C, Devaney E"
image: /assets/images/papers/2022-06-13-laing-xqtl-rnaseq.png
redirect_from: 
fulltext: "https://doi.org/10.1371/journal.ppat.1010545"
pdflink:
github:
pmid: 
pmcid: 
f1000:
biorxiv: "https://www.biorxiv.org/content/10.1101/2021.11.12.468372v2" 
doi:
dryad_doi:
figshare_doi: 
zenodo_doi: 
altmetric_id: 
category: paper
# Note: 'published' is a Jekyll keyword and does not refer to whether the paper is published, but rather to whether this Markdown should be part of the rendered site.
published: true
preprint: true
embargo: false	
peerreview: false
review: false
tags: [hcontortus, x-qtl, drug resistance, genetic mapping]
---
{% include JB/setup %}

# Abstract 

The antiparasitic drug ivermectin plays an essential role in human and animal health globally. However, ivermectin resistance is widespread in veterinary helminths and there are growing concerns of sub-optimal responses to treatment in related helminths of humans. Despite decades of research, the genetic mechanisms underlying ivermectin resistance are poorly understood in parasitic helminths. This reflects significant uncertainty regarding the mode of action of ivermectin in parasitic helminths, and the genetic complexity of these organisms; parasitic helminths have large, rapidly evolving genomes and differences in evolutionary history and genetic background can confound comparisons between resistant and susceptible populations. We undertook a controlled genetic cross of a multi-drug resistant and a susceptible reference isolate of Haemonchus contortus, an economically important gastrointestinal nematode of sheep, and ivermectin-selected the F2 population for comparison with an untreated F2 control. RNA-seq analyses of male and female adults of all populations identified high transcriptomic differentiation between parental isolates, which was significantly reduced in the F2, allowing differences associated specifically with ivermectin resistance to be identified. In all resistant populations, there was constitutive upregulation of a single gene, HCON_00155390:cky-1, a putative pharyngeal-expressed transcription factor, in a narrow locus on chromosome V previously shown to be under ivermectin selection. In addition, we detected sex-specific differences in gene expression between resistant and susceptible populations, including constitutive upregulation of a P-glycoprotein, HCON_00162780:pgp-11, in resistant males only. After ivermectin selection, we identified differential expression of genes with roles in neuronal function and chloride homeostasis, which is consistent with an adaptive response to ivermectin-induced hyperpolarisation of neuromuscular cells. Overall, we show the utility of a genetic cross to identify differences in gene expression that are specific to ivermectin selection and provide a framework to better understand ivermectin resistance and response to treatment in parasitic helminths.

## Data availability

Fastq files of trimmed reads have been submitted to ENA under study accession
PRJEB4207. The *H. contortus* V4 genome and annotations (version WBPS15) are
available at WormBase Parasite https://parasite.wormbase.org/Haemonchus_contortus_prjeb506.
