---
layout: paper
title: "Evaluation of genome skimming to detect and characterise human and livestock helminths"
year: "2022"
shortref: "Papaiakovou <i>et al.</i> IJP 2022"
nickname: genome-skimming
journal: "International Journal for Parasitology"
volume:
issue:
pages:
authors: "Papaiakovou M, Fraija-Fern√°ndez N, James K, Briscoe AG, Hall A, Jenkins TP, Dunn J, Levecke B, Mekonnen Z, Cools P, Doyle SR, Cantacessi C, Littlewood DTJ "
image: /assets/images/papers/
redirect_from:
fulltext:
pdflink:
github:
pmid:
pmcid:
f1000:
doi: DOI
dryad_doi:
figshare_doi:
zenodo_doi:
altmetric_id:
category: paper
# Note: 'published' is a Jekyll keyword and does not refer to whether the paper is published, but rather to whether this Markdown should be part of the rendered site.
published: true
preprint: true
embargo: false
peerreview: false
review: false
tags: [STH, genome-skimming, genomics]
---
{% include JB/setup %}

# Abstract

The identification of gastrointestinal helminth infections of humans and livestock almost exclusively relies on the detection of eggs or larvae in faeces, followed by manual counting and morphological characterisation to differentiate species using microscopy-based techniques. However, molecular approaches based on the detection and quantification of parasite DNA are becoming more prevalent, increasing the sensitivity, specificity and throughput of diagnostic assays. High-throughput sequencing, from single PCR targets through to the analysis of whole genomes, offers significant promise towards providing information-rich data that may add value beyond traditional and conventional molecular approaches; however, thus far, its utility has not been fully explored to detect helminths in faecal samples. In this study, low-depth whole genome sequencing, i.e. genome skimming, has been applied to detect and characterise helminth diversity in a set of helminth-infected human and livestock faecal material. The strengths and limitations of this approach is evaluated using three methods to characterise and differentiate metagenomic sequencing data based on (i) mapping to whole mitochondrial genomes, (ii) whole genome assemblies, and (iii) a comprehensive ITS2 database, together with validation using quantitative PCR (qPCR). Our analyses suggest that genome skimming can successfully identify most single and multi-species infections reported by qPCR and can provide sufficient coverage within some samples to resolve consensus mitochondrial genomes, thus facilitating phylogenetic analyses of selected genera, e.g. Ascaris spp. Key to this approach is both the availability and integrity of helminth reference genomes, some of which are currently contaminated with bacterial and host sequences. The success of genome skimming of faecal DNA is dependent on the availability of vouchered sequences of helminths spanning both taxonomic and geographic diversity, along with methods to detect or amplify minute quantities of parasite nucleic acids in mixed samples.
