---
layout: paper
title: "Population genomics of ancient and modern <i>Trichuris trichiura</i>"
year: "2022"
shortref: "Doyle <i>et al.</i> <i>Nat Commun</i> 2022"
nickname: ancient-trichuris
journal: "Nature Communications"
volume: 
issue:
pages: 
authors: "Doyle SR, Soe MJ, Nejsum P, Betson M, Cooper PJ, Peng L, Zhu XQ, Sanchez A, Matamoros G, Fontecha Sandoval GA, Cutillas C, Tchuente LAT, Mekonnen Z, Ame SM, Namwanje H, Levecke B, Berriman M, Fredensborg BL, Kapel CMO"
image: /assets/images/papers/2022-07-06-doyle-ancient-trichuris.png
redirect_from: 
fulltext: https://doi.org/10.1038/s41467-022-31487-x
pdflink: 
github: https://github.com/stephenrdoyle/ancient_trichuris
pmid: 
pmcid: 
f1000: 
doi: "https://doi.org/10.1038/s41467-022-31487-x"
biorxiv: "https://doi.org/10.1101/2021.10.21.464505"
dryad_doi:
figshare_doi: 
zenodo_doi: 
altmetric_id: 
category: paper
# Note: 'published' is a Jekyll keyword and does not refer to whether the paper is published, but rather to whether this Markdown should be part of the rendered site.
published: true
preprint: true
embargo: false	
peerreview: false
review: false
tags: [ttrichuris, population genomics, assembly, ancient]
---
{% include JB/setup %}

# Abstract 
The neglected tropical disease trichuriasis is caused by the whipworm <i>Trichuris trichiura</i>, a soil-transmitted helminth that has infected humans for millennia. Today, <i>T. trichiura</i> infects as many as 500 million people, predominantly in communities with poor sanitary infrastructure enabling sustained faecal-oral transmission. Using whole-genome sequencing of geographically distributed worms collected from human and other primate hosts, together with ancient samples preserved in archaeologically-defined latrines and deposits dated up to one thousand years old, we present the first population genomics study of <i>T. trichiura</i>. We describe the continent-scale genetic structure between whipworms infecting humans and baboons relative to those infecting other primates. Admixture and population demographic analyses support a stepwise distribution of genetic variation that is highest in Uganda, consistent with an African origin and subsequent translocation with human migration. Finally, genome-wide analyses between human samples and between human and non-human primate samples reveal local regions of genetic differentiation between geographically distinct populations. These data provide insight into zoonotic reservoirs of human-infective <i>T. trichiura</i> and will support future efforts toward the implementation of genomic epidemiology of this globally important helminth.


## Data availability
Raw sequencing data is available under the European Nucleotide Archive (ENA) study accession “ERP128004”. Individual ENA sample accessions are described in Supplementary Data 1. The T. trichiura reference genome and assembled mitochondrial genomes are available from https://github.com/stephenrdoyle/ancient_trichuris/tree/master/02_data. The publicly available mitochondrial genomes used for comparative analysis (Accession numbers: “KT449826”, “GU385218“, “HG806815“, “KT449822“, “KT449823“, “GU070737“, “JQ996231“, “JQ996232“, “KC461179“, “NC_017747”, “NC_018597”, “NC_018596”, “NC_028621”, “NC_002681”) are available from ENA and NCBI. There are no restrictions on data availability.

## Code availability
Custom code to analyse data and reproduce the figures presented is available at https://stephenrdoyle.github.io/ancient_trichuris/ and is archived under a stable DOI at Zenodo.

