---
layout: paper
title: "Genome structure and population genomics of the canine heartworm Dirofilaria immitis"
year: "2023"
shortref: "Gandasegui <i>et al.</i> BioRxiv 2023"
nickname: heartworm-genome-popgen
journal: "BioRxiv"
volume:
issue:
pages:
authors: "Gandasegui J, Power RI, Curry E, Lau DCW, O'Neill CM, Wolstenholme A, Prichard R, Slapeta J, Doyle SR"
image: /assets/images/papers/2023-04-26-gandasegui-dirofilaria-genomics.png
redirect_from:
fulltext: https://www.biorxiv.org/content/10.1101/2023.04.25.538225v1
pdflink:
github: https://github.com/Gandasegui/pop_genomics_dirofiliaria
pmid:
pmcid:
f1000:
doi: https://doi.org/10.1101/2023.04.25.538225
dryad_doi:
figshare_doi:
zenodo_doi:
altmetric_id:
category: paper
# Note: 'published' is a Jekyll keyword and does not refer to whether the paper is published, but rather to whether this Markdown should be part of the rendered site.
published: true
preprint: true
embargo: false
peerreview: false
review: false
tags: [dimmitis, genome, population genetics]
---
{% include JB/setup %}

# Abstract

The heartworm, *Dirofilaria immitis*, is a filarial parasitic nematode responsible for significant morbidity and mortality in wild and domesticated canids. Resistance to macrocyclic lactone drug prevention represents a significant threat to parasite control and has prompted investigations to understand the genetic determinants of resistance. This study aimed to improve the genomic resources of *D. immitis* to enable a more precise understanding of how genetic variation is distributed within and between parasite populations worldwide, which will inform the likelihood and rate by which parasites, and in turn, resistant alleles, might spread. We have guided the scaffolding of a recently published genome assembly for *D. immitis* (ICBAS_JMDir_1.0) using the chromosomal-scale reference genomes of *Brugia malayi* and *Onchocerca volvulus*, resulting in an 89.5 Mb assembly composed of four autosomal- and one X-linked chromosomal-scale scaffolds representing 99.7% of the genome. Publicly available and new whole-genome sequencing data from 32 *D. immitis* samples from Australia, Italy and the USA were assessed using principal component analysis, nucleotide diversity (Pi) and absolute genetic divergence (Dxy) to characterise the global genetic structure and measure within- and between population diversity. These population genetic analyses revealed broad-scale genetic structure among globally diverse samples and differences in genetic diversity between populations; however, fine-scale subpopulation analysis was limited and biased by differences between sample types. Finally, we mapped SNPs previously associated with macrocyclic lactone resistance in the new genome assembly, revealing physical linkage of high-priority variants on chromosome 3, and determined their frequency in the studied populations. This new chromosomal assembly for *D. immitis* now allows for a more precise investigation of selection on genome-wide genetic variation and will enhance our understanding of parasite transmission and the spread of genetic variants responsible for resistance to treatment.
